### PR TITLE
Release notes for 4.25.1

### PR DESCRIPTION
**Pull request by lorenrh**

### CODE REVIEW

File: content/desktop/release-notes.md [hunk 1]
```diff
@@ -24,6 +24,28 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+## 4.25.1
+
+{{< release-date date="2023-10-13" >}}
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:29]: 2023-11-09 or 2023-11-13?

File: content/desktop/release-notes.md [hunk 2]
```diff
@@ -24,6 +24,28 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+
+{{< desktop-install all=true version="4.25.1" build_path="/" >}}
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Set sysctl vm.max_map_count to 262144. See [docker/for-mac#7047](https://github.com/docker/for-mac/issues/7047)
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:37]:
```suggestion
- The `sysctl vm.max_map_count` is now set to 262144. See [docker/for-mac#7047](https://github.com/docker/for-mac/issues/7047)
```

File: content/desktop/release-notes.md [hunk 3]
```diff
@@ -24,6 +24,28 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+
+#### For Mac
+
+- Fixed a bug when swap is disabled. Fixes [docker/for-mac#7045](https://github.com/docker/for-mac/issues/7045) and [docker/for-mac#7044](https://github.com/docker/for-mac/issues/7044).
+
+#### For Windows
+
+- Fixed an issue where **Switch to Windows Containers** would not appear on the tray menu for some users. See [docker/for-win#13761](https://github.com/docker/for-win/issues/13761).
+- Fixed a bug where WSL integration would not work for users using a shell other than `sh`. See [docker/for-win#13764](https://github.com/docker/for-win/issues/13764).
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:46]:
```suggestion
- Fixed a bug where the WSL integration would not work for users using a shell other than `sh`. See [docker/for-win#13764](https://github.com/docker/for-win/issues/13764).
```

File: content/desktop/release-notes.md (hunk 1 above)
Comment (aevesdocker) [inline content/desktop/release-notes.md:29]:
```suggestion
{{< release-date date="2023-11-13" >}}
```

**Review by aevesdocker — DISMISSED** [review]
✨

File: content/desktop/release-notes.md [hunk 4]
```diff
@@ -24,6 +24,28 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- The `sysctl vm.max_map_count` is now set to 262144. See [docker/for-mac#7047](https://github.com/docker/for-mac/issues/7047)
+
+#### For Mac
+
+- Fixed a bug when swap is disabled. Fixes [docker/for-mac#7045](https://github.com/docker/for-mac/issues/7045) and [docker/for-mac#7044](https://github.com/docker/for-mac/issues/7044).
```
Comment (MihaelaStoica) [inline content/desktop/release-notes.md:41]: This is a fix for all platforms, so it needs to be moved in the "For all platforms" section. I also added a suggestion with the for-win issue.

Comment (MihaelaStoica) [inline content/desktop/release-notes.md:41]:
```suggestion
- Fixed a bug when swap is disabled. Fixes [docker/for-mac#7045](https://github.com/docker/for-mac/issues/7045), [docker/for-mac#7044](https://github.com/docker/for-mac/issues/7044) and [docker/for-win#13779](https://github.com/docker/for-win/issues/13779).
```

**Comment by MihaelaStoica** [comment]
We seems to be missing the release note from https://github.com/docker/pinata/pull/25076

**Comment by lorenrh** [comment]
> We seems to be missing the release note from [docker/pinata#25076](https://github.com/docker/pinata/pull/25076)


oh my bad! Definitely missed it 🙇  re-added it now. I'm guessing that is part of **For all platforms**, right?

**Review by aevesdocker — APPROVED** [review]

**Pull request merged**